string canPort
uint32 canID
---
bool res